app/src/main/java/com/aerogear/androidshowcase/di/SecureApplicationModule.java

```diff
@@ -91,12 +91,10 @@ AuthService provideAuthService(Context context) {
                         //be configured.  We are not returning an Optional because we can't guarantee
                         //Optional is available on Android M and L and we don't want to add Guava.
         }
-        AuthService authService = core.getService(AuthService.class);
         AuthServiceConfiguration authServiceConfig = new AuthServiceConfiguration.AuthConfigurationBuilder()
                 .withRedirectUri("com.aerogear.androidshowcase:/callback")
                 .build();
-
-        authService.init(context, authServiceConfig);
+        AuthService authService = new AuthService(authServiceConfig);
         return authService;
```

Comment:
This could be changed to `return new AuthService(authServiceConfig);`

Reply:
Done!